assessment.sql

```diff
@@ -5,24 +5,44 @@ CREATE DATABASE "sql-crud-assessment";
 
 -- Question 1. Create a table called `licensees` to hold data in columns surname,given_name,gender,height,weight,dob,eye_color,hair_color.  See `data/licensees.csv` for example data.  Don't forget that all tables should have an id column as a unique identifier for a row.
 
--- your code here
+CREATE TABLE licensees(
+   id SERIAL PRIMARY KEY,
+   surname CHARACTER VARYING,
+   given_name CHARACTER VARYING,
+   gender CHARACTER VARYING(1),
+   height NUMERIC,
+   weight NUMERIC,
+   dob DATE,
+   eye_color CHARACTER VARYING,
+   hair_color CHARACTER VARYING
+);
 
 -- Question 2. Add `Bishop, Caren, f, 63, 132, 1943-09-26, Brown, Black` into the table of licensees.
 
--- your code here
+INSERT INTO licensees (surname,given_name,gender,height,weight,dob,eye_color,hair_color)
+VALUES ('Bishop', 'Caren', 'f', 63, 132, '1943-09-26', Brown, Black);
```

Comment:
`Brown` and `Black` should be strings, not identifiers. See what you did with `'Bishop`' and `'Caren'`. 9/10